### ISSUE TITLE
Support for default verb?

### ISSUE THREAD

**Issue by alexreg** [gsscoder/commandline#494]
Is there any support for some sort of "default" verb? That is, a verb that gets executed when no verb is actually specified. Perhaps an existing verb can be marked as default, even?

**Comment by alexreg** [comment]
Furthermore, is it possible to have global options while still having a verb?

**Comment by nemec** [comment]
This is a duplicate of #490. Unfortunately, no it isn't possible. With the way verbs work in the current version (e.g. `Parse<VerbA, VerbB>(args)`) I'm having a hard time thinking of a way to add a default verb and global options.

**Comment by alexreg** [comment]
Thanks for the quick reply. What about a simple `IsDefault` property of the `Verb` attribute. The name of the verb should also be optional in this case. As for global options, what about a special Options generic argument that inherits from a (dummy) GlobalOptions class, or something similar?

**Comment by alexreg** [comment]
I'm currently trying to come up with a workaround for this, but I can't get the `NoVerbSelectedError`, nor the `BadVerbSelectedError`. Using the latest beta here.

**Comment by alexreg** [comment]
Update: it seems I have to specify at least *two* options types to `ParseArguments` for these errors to kick in! I see why, though perhaps this should be documented.

**Comment by nemec** [comment]
You're right - it is not obvious that one generic argument is interpreted as an Options class but multiple are considered Verbs.

**Comment by TheFanatr** [comment]
Just to clarify, is there a workaround by using GlobalOptions?
